docs/devel/collab.md

```diff
@@ -28,7 +24,7 @@ Maintainers can assign reviews to other maintainers, when appropriate. The assig
 
 ## Merge hours
 
-Maintainers will do merges between the hours of 7:00 am Monday and 7:00 pm (19:00h) Friday.  PRs that arrive over the weekend or on holidays will only be merged if there is a very good reason for it and if the code review requirements have been met.
+Maintainers will do merges during business days (according to US calendar). PRs that arrive over the weekend or on holidays will only be merged if there is a very good reason for it and if the code review requirements have been met.
```

Comment:
This reads as the same as what you changed it from, but less precisely.  How about:

```
Maintainers will do merges of appropriately reviewed-and-approved changes during their local "business hours" (typically 7:00 am Monday to 5:00 pm (17:00h) Friday).  PRs that arrive over the weekend or on holidays will only be merged if there is a very good reason for it and if the code review requirements have been met.  Concretely this means that nobody should merge changes immediately before going to bed for the night.
```